ui/src/containers/NodePageMetricsTab.js

```diff
@@ -147,36 +336,72 @@ const NodePageMetricsTab = (props) => {
     title: null,
   };
 
-  const colorUsage = {
+  const strokeDashConfig = {
+    field: 'symbol',
+    type: 'nominal',
+    legend: {
+      direction: 'horizontal',
+      orient: 'bottom',
+      title: null,
+      values: ['Cluster avg'],
+      symbolSize: 300,
+      labelFontSize: 15,
```

Comment:
I feel the font size of the legend is a bit big, even bigger than the graph title, so I suggest a smaller size. And please apply to all legends.
```suggestion
      labelFontSize: 14,
```